AudibleApi/Authorization/IdentityMaintainer.cs

```diff
@@ -43,9 +41,9 @@ public static async Task<IdentityMaintainer> CreateAsync(IIdentity identity, IAu
 
 		protected IdentityMaintainer(IIdentity identity, IAuthorize authorize, ISystemDateTime systemDateTime)
 		{
-			_identity = identity ?? throw new ArgumentNullException(nameof(identity));
-			_authorize = authorize ?? throw new ArgumentNullException(nameof(authorize));
-			SystemDateTime = systemDateTime ?? throw new ArgumentNullException(nameof(systemDateTime));
+			_identity = ArgumentValidator.EnsureNotNull(identity, nameof(identity));
```

Comment:
Doesn't it feel good to simplify boilerplate like this?

Reply:
Very! 😂